packages/postgres/src/default-pool.ts

```diff
@@ -132,19 +129,30 @@ export const callWithClientZeroOrOneRowAsync =
   defaultPool.callWithClientZeroOrOneRowAsync.bind(defaultPool);
 export const queryRows = defaultPool.queryRows.bind(defaultPool);
 /**
- * Wrapper around {@link queryOneRowAsync} that parses the resulting row with
- * the given Zod schema. If the query doesn't return exactly one row, an error
- * is thrown.
+ * Executes a query with the specified parameters. Errors if the query does
+ * not return exactly one row that conforms to the given Zod schema.
+ *
  */
 export const queryRow = defaultPool.queryRow.bind(defaultPool);
 /**
- * Wrapper around {@link queryZeroOrOneRowAsync} that parses the resulting row
- * (if any) with the given Zod schema. Returns either null or a single row, and
- * errors otherwise.
+ * Executes a query with the specified parameters. Returns either null or a
+ * single row that conforms to the given Zod schema, and errors otherwise.
  */
 export const queryOptionalRow = defaultPool.queryOptionalRow.bind(defaultPool);
+/**
+ * Calls the given function with the specified parameters.
```

Comment:
I think "function" here is overloaded, maybe use "sproc" to clarify this refers to a database stored procedure?

Reply:
Done in [7ecd4be](https://github.com/PrairieLearn/PrairieLearn/pull/12437/commits/7ecd4be75a6ac88c7e0763977286c0f90f5c41fc).